src/Sentry/Internal/Http/CachingTransport.cs

```diff
@@ -143,17 +146,9 @@ private void EnsureFreeSpaceInCache()
 
         private IEnumerable<string> GetCacheFilePaths()
         {
-            try
-            {
-                return Directory
-                    .EnumerateFiles(_isolatedCacheDirectoryPath, $"*.{EnvelopeFileExt}")
-                    .OrderBy(f => new FileInfo(f).CreationTimeUtc);
-            }
-            catch (DirectoryNotFoundException)
-            {
-                _options.LogWarning("Cache directory is empty.");
-                return Array.Empty<string>();
-            }
+            return Directory
```

Comment:
I'd suggest going  with 
```csharp
        private IEnumerable<string> GetCacheFilePaths()
            => Directory
                  .EnumerateFiles(_isolatedCacheDirectoryPath, $"*.{EnvelopeFileExt}")
                  .OrderBy(f => new FileInfo(f).CreationTimeUtc);
```
Since it's a "single line" function.